### ISSUE TITLE
iron-component-page accessibility issues

### ISSUE THREAD

**Issue by lpalmaro** [PolymerElements/iron-component-page#15]
Keyboard focus: I am unable to get to the show private API / collapse / expand controls using just the keyboard. 

Once we fix the keyboard navigation issues here, please make sure that focus is visible for these controls. 

Also, there currently doesn't appear to be any content in the drop down menu in the upper left hand corner, but please test that this works with keyboard (e.g. you can navigate to the menu with tab, press enter to open, use teh arrows to go through the contents, and escape to close). 

Contrast: contrast on show private API is poor - please darken that text.

**Comment by lpalmaro** [comment]
Also, here are the audit results: 
[Severe] Controls and media elements should have labels (1)
[Warning] Text elements should have a reasonable contrast ratio (3)
[Warning] The web page should have a title that describes topic or purpose (1)
Not applicable tests (11)
Passing tests (3)

**Comment by atotic** [comment]
Added keyboard focus to show private API by replacing it with paper-button.
Drop down menu is navigable 
Contrast on show/hide api is good now that it is a button.
Page gets a default title from element name if none was set.
Fixed in 518de6b
Thansk for the report.
